src/Shared/LoggingHelpers.cs

```diff
@@ -17,7 +17,10 @@ public static void WriteDashboardUrl(ILogger logger, string? dashboardUrls, stri
 
         if (StringUtils.TryGetUriFromDelimitedString(dashboardUrls, ";", out var firstDashboardUrl))
         {
-            logger.LogInformation("Login to the dashboard at {DashboardLoginUrl}", $"{firstDashboardUrl.GetLeftPart(UriPartial.Authority)}/login?t={token}");
+            var message = !isContainer
+                ? "Login to the dashboard at {DashboardLoginUrl}"
+                : "Login to the dashboard at {DashboardLoginUrl}. The URL may need changes depending on how network access to the container is configured.";
+            logger.LogInformation(message, $"{firstDashboardUrl.GetLeftPart(UriPartial.Authority)}/login?t={token}");
```

Comment:
We should consider localizing this message.

Reply:
We're not localizing any console messages at the moment. If we do, it should be done holistically.